erdpy/ledger/config.py

```diff
@@ -1,12 +1,12 @@
-class ElrondLedgerAppConfiguration:
+class MultiversXLedgerAppConfiguration:
```

Comment:
Unfortunately, this is a breaking change - in theory (some developers might have a reference to this class).

In practice, perhaps it is not.

@bogdan-rosianu, @ccorcoveanu, can we keep the old name for a bit more? Then apply the renaming in erdpy v4?